e2e-test/app_helper.py

```diff
@@ -35,32 +35,41 @@ def poll_for_app_url(workspaceId, app_type, proxy_url_name, azure_token, leo_url
 
     # prevent infinite loop
     poll_count = 20 # 30s x 20 = 10 min
+    
+    for i in range(0,3):
+        while True:
```

Comment:
Why do you need both the `while True` _and_ the for loop?

Reply:
+1

Reply:
ha I thought I had a great reason to use it but when I wrote it down I realized actually it is not needed 😆